drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
```

Comment:
Though the Remote driver implementation is not complete, there must be a way for the actual remote drivers to return the type they support. This is ofcourse required for the `Type()` call.
What is the alternate way proposed in this PR to provide that info ?